packages/bot/src/transformers/types.ts

```diff
@@ -1768,6 +1768,8 @@ export interface User {
   verified: boolean
   /** data for the user's collectibles */
   collectibles?: Collectibles
+  /** The user's primary guild */
+  primaryGuild?: UserPrimaryGuild | null
```

Comment:
```suggestion
  primaryGuild?: UserPrimaryGuild
```

If I recall correctly, there was a discussion on preserving null values, but for now it should be optional